tests/fpga_stencil.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
```

Comment:
Don't we have a `tests/fpga` subfolder for this?

Reply:
Nope, only Xilinx/Intel-specific ones